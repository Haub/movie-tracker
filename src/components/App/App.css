html {
  background-image: url("../../images/concrete.jpg");
  background-repeat: repeat; }
