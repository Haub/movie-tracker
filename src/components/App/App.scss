$mainYellow: #f9f14d;

html {
  background-image: url('../../images/brick.jpg');
  background-repeat: repeat;
}
.App {
  text-align: center;
}

body {
  margin: 0;
}

.title {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  margin: 0 auto;
  border: none;
  font: normal 82px/normal "Anton", Helvetica, sans-serif;
  color: rgb(112, 112, 112);
  text-align: left;
  text-transform: uppercase;
  font-size: 2.5rem;
  text-overflow: clip;
  letter-spacing: 3px;
}

