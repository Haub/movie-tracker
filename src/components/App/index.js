import React, { Component } from 'react';

import ContentRoute from '../../containers/ContentRoute/';
import FormRoute from '../../containers/UserInputForm/FormRoute';
import './App.css';

class App extends Component {

  render() {
    return (
      <div className="App">
        <header className="header">
          <FormRoute />
          <h1 className="title">Hanks Movies</h1>
        </header>
        <ContentRoute />
      </div>
    );
  }
}

export default App;