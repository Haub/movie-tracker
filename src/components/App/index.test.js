import React from 'react';
import { shallow } from 'enzyme';

import App from '.';

it('matches snapshot', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
