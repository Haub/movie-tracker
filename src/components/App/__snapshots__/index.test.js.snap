// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <header
    className="header"
  >
    <h1
      className="title"
    >
      HANKY PANKY
    </h1>
    <withRouter() />
  </header>
  <withRouter(Connect(ContentRoute)) />
</div>
`;
